gtsfm/multi_view_optimizer.py

```diff
@@ -85,9 +88,8 @@ def create_computation_graph(
 
         multiview_optimizer_metrics_graph = [averaging_metrics, data_assoc_metrics_graph, ba_metrics_graph]
 
-        if gt_poses_graph is not None:
-            # align the sparse multi-view estimate before BA to the ground truth pose graph.
-            ba_input_graph = dask.delayed(ba_input_graph.align_via_Sim3_to_poses)(gt_poses_graph)
+        # align the sparse multi-view estimate before BA to the ground truth pose graph.
+        ba_input_graph = dask.delayed(ba_input_graph.align_via_Sim3_to_poses)(gt_poses_graph)
```

Comment:
dont we only want to do this if the GT exists?

Reply:
We will only reach this line if gt_cameras_graph is not None